.tool-versions

```diff
@@ -0,0 +1,7 @@
+poetry 1.1.6
```

Comment:
Two things
- Doesn't this change also require changes to all CI workflows to add the asdf installation step
- asdf is officially Linux and Mac only, this issue says it might work on wsl2 https://github.com/asdf-vm/asdf/issues/788

Reply:
@phanirithvij2000 not sure what you mean—the `asdf` change is for local development only and not meant to modify CI at all.

Reply:
Seems like I misunderstood the purpose of this PR sorry about this.